src/patternfly/components/ClipboardCopy/clipboard-copy-group-copy.hbs

```diff
@@ -0,0 +1,8 @@
+<button class="pf-c-clipboard-copy__group-copy{{#if clipboard-copy-group-copy--modifier}} {{clipboard-copy-group-copy--modifier}}{{/if}}"
+  aria-label="copy to clipboard"
```

Comment:
I think the label of this button should have a reference to the label applied to the input. I tested a couple of variations, and the following work:

Variation 1 - no visible or hidden label for input:
- add `id` and `aria-labelledby="[id of copy button] [id of input]"`
- `<input>` has `aria-label` as you have in your example

Variation 2 - a visible or hidden label for input is defined:
- add `id` and `aria-labelledby="[id of copy button] [id of label]"`
  - It's important to note that the id for the label must be used in this case, or the JAWS will announce the value of the input and not the label
- add `id` to the `<label>`

Variation 3 - the input label is repeated in the Copy button label
- `aria-label` for the button would be something like "Copy to clipboard: URL" assuming "URL is the text input label

I think it's fine to just show Variation 1 in your examples. But I would include something like the following for the `aria-labelledby` attribute documentation:


Attribute | Applied To | Outcome
-- | -- | --
`aria-labelledby="[id of button] [id of input label]"` | `.pf-c-clipboard-copy__group-copy` | Provides an accessible name that is unique and communicates context of the button. Required when more than one ClipboardCopy component exists on the page. **Important:** If the label is defined on the `<input>` using `aria-label`, then use the `id` of the `<input>`. If the label is defined in a `<label>`, then use the `id` of the `<label>`. **Alternatively** this attribute can be ignored if the text input label is defined as part of the value in `aria-label`.

Reply:
@seanforyou23 Would you review this previous comment and let me know if you would suggest something different?

Also, @christiemolloy feel free to revise the documentation I posted.

Reply:
Thanks for this going with variation 1.